geode-core/src/test/java/org/apache/geode/internal/cache/partitioned/rebalance/PartitionedRegionLoadModelJUnitTest.java

```diff
@@ -192,15 +199,15 @@ public void testRedundancySatisfactionDoNotEnforceLocalMaxMemory() throws Except
    * with remote IP addresses.
    */
   @Test
-  public void testRedundancySatisfactionPreferRemoteIp() throws Exception {
+  public void testRedundancySatisfactionPreferRemoteIp() throws UnknownHostException {
     PartitionedRegionLoadModel model = new PartitionedRegionLoadModel(bucketOperator, 1, 3,
-        getAddressComparor(false), Collections.<InternalDistributedMember>emptySet(), null);
+        getAddressComparor(false), Collections.emptySet(), null);
     InternalDistributedMember member1 =
         new InternalDistributedMember(InetAddress.getByName("127.0.0.1"), 1);
     InternalDistributedMember member2 =
         new InternalDistributedMember(InetAddress.getByName("127.0.0.1"), 2);
     InternalDistributedMember member3 =
-        new InternalDistributedMember(InetAddress.getLocalHost(), 3);
+        new InternalDistributedMember(SocketCreator.getLocalHost(), 3);
```

Comment:
for those looking this is the line change that is for not being dependent on `/etc/hosts`